content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -28,6 +28,10 @@ from dvclive import Live
 
 live = Live("training_metrics")
 
+live.log_param("num_epochs", NUM_EPOCHS)
```

Comment:
I suggest not touching the `DVCLive with DVC` page until we complete some more steps to better integrate it into DVC. I don't think it makes much sense right now under stage params.

Reply:
Instead, I think this material should be added to `Get Started`.

Reply:
Agree